### PR TITLE
Invoice items

### PR DESCRIPTION
**Pull request by Kledin85**

### CODE REVIEW

File: lib/invoice_item.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require 'bigdecimal'
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/invoice_item.rb:1]: Ignore

File: lib/invoice_item.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+class InvoiceItem
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/invoice_item.rb:3]: ignore

File: lib/invoice_item_repository.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+require_relative '../lib/modules/repo_queries'
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/invoice_item_repository.rb:1]: ignore

File: lib/invoice_item_repository.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+require 'bigdecimal'
+
+class InvoiceItemRepository
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/invoice_item_repository.rb:4]: ignore

**Review by Pocketzs — CHANGES_REQUESTED** [review]

**Review by beddings81 — APPROVED** [review]

**Review by Pocketzs — APPROVED** [review]

**Review by KaraJoHo — APPROVED** [review]

**Pull request merged**